packages/api/endpoints/granules.js

```diff
@@ -561,14 +564,22 @@ async function delByGranuleId(req, res) {
     }
   }
 
-  await deleteGranuleAndFiles({
+  try {
+    dynamoGranule = await granuleModelClient.getRecord({ granuleId });
```

Comment:
Check this. I don't think we should have any references to dynamo in this the rds-phase-3 branch so if they're being added back here, it seems like a mistake.